presto-mongodb/src/main/java/com/facebook/presto/mongodb/MongoSession.java

```diff
@@ -117,14 +121,13 @@ public MongoSession(TypeManager typeManager, MongoClient client, MongoClientConf
 
         this.tableCache = CacheBuilder.newBuilder()
```

Comment:
Make sure to add a test with multiple tables that differ only with case -- this cache builder looks like it would overwrite the values rather than preserving duplicates